inter/ibr/inter_block_records.go

```diff
@@ -10,21 +10,21 @@ import (
 )
 
 type LlrBlockVote struct {
-	Atropos      hash.Event
-	Root         hash.Hash
+	BlockHash    hash.Hash
+	StateRoot    hash.Hash
 	TxHash       hash.Hash
 	ReceiptsHash hash.Hash
 	Time         inter.Timestamp
 	GasUsed      uint64
 }
 
 type LlrFullBlockRecord struct {
-	Atropos  hash.Event
-	Root     hash.Hash
-	Txs      types.Transactions
-	Receipts []*types.ReceiptForStorage
-	Time     inter.Timestamp
-	GasUsed  uint64
+	BlockHash hash.Hash
+	StateRoot hash.Hash
+	Txs       types.Transactions
+	Receipts  []*types.ReceiptForStorage
+	Time      inter.Timestamp
+	GasUsed   uint64
 }
```

Comment:
Note: LlrBlockVote/LlrFullBlockRecord structs are relevant only for legacy Opera mainnet events support.
If we would break their compatibility (the changes above does not), they can be removed immediately.